### ISSUE TITLE
DEPRECATION: `ember-cli/ext/promise`

### ISSUE THREAD

**Issue by noxan** [dschmidt/ember-cli-deploy-sentry#34]
```
DEPRECATION: `ember-cli/ext/promise` is deprecated, use `rsvp` instead. Required here: 
  Object.<anonymous> (node_modules/ember-cli-deploy-sentry/index.js:4:17)
```

see https://github.com/ember-cli-deploy/ember-cli-deploy/issues/465

**Comment by begedin** [comment]
I forgot to create a reference, but this should be resolved by #35

**Comment by noxan** [comment]
Wow, that's been fast... I close the issue since the PR has been merged.

**Comment by begedin** [comment]
@noxan Keep in mind it's still on master. You'll probably want an official release to, so you can update.

**Comment by dschmidt** [comment]
I just released v0.5.2. Let me know if it works.

**Comment by noxan** [comment]
Can confirm the deprecation being resolved. Many thanks for the fast patch and release. Great job guys, keep it up!
